app/views/home/index.html.erb

```diff
@@ -0,0 +1,87 @@
+<div class="govuk-grid-row subnav">
+  <div class="govuk-grid-column-one-third">
+    <h3 class="govuk-heading-m"><%= @name %></h3>
+  </div>
+
+  <nav class="govuk-grid-column-two-thirds">
+    <%= link_to "Overview", '', class: "govuk-link active" %>
+    <%= link_to "Team Members", team_members_path, class: "govuk-link" %>
+    <%= link_to "Settings", ips_path, class: "govuk-link" %>
+  </nav>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Get GovWifi in your building</h2>
+
+    <h3 class="govuk-heading-m"> 1. Configure your authenticator(s) </h3>
```

Comment:
Is the whitespace within the html tags on purpose?
Seeing quite a lot of this.

Reply:
Oh I see this was copied over from the previous markup.  Bizarre, need to fix this but can do in a later PR

Reply:
Yeah, not on purpose. Want a linter or something to catch this stuff.